src/chain/sync.ts

```diff
@@ -234,6 +253,10 @@ export default class ChainSync {
 
 			const start = Date.now()
 			if (this._state == State.RUNNING) {
+				if (block.number != this.lastBlock.blockNo + 1)
+					Logging.showError(
```

Comment:
This should probably bee showLogError since it shouldn't occur, but if it does (and you happen to be running with log enabled), it should be recorded.

Reply:
#4 Refers